### ISSUE TITLE
OS 2.0 doesn't start on Fedora 24/Python 3.5 (module 'html5lib.treebuilders' has no attribute '_base')

### ISSUE THREAD

**Issue by maweki** [OpenSlides/OpenSlides#2585]
I tried the installation instructions, creating a virtual environment and installing Openslides with pip. Trying to start openslides I get the following:

```
(.virtualenv) mwenzel@mwenzel-desktop:/home/mwenzel/src/OpenSlides
$ openslides start
Traceback (most recent call last):
  File "/home/mwenzel/src/OpenSlides/.virtualenv/bin/openslides", line 11, in <module>
    sys.exit(main())
  File "/home/mwenzel/src/OpenSlides/.virtualenv/lib64/python3.5/site-packages/openslides/__main__.py", line 41, in main
    known_args.callback(known_args)
  File "/home/mwenzel/src/OpenSlides/.virtualenv/lib64/python3.5/site-packages/openslides/__main__.py", line 176, in start
    execute_from_command_line(['manage.py', 'migrate'])
  File "/home/mwenzel/src/OpenSlides/.virtualenv/lib64/python3.5/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/home/mwenzel/src/OpenSlides/.virtualenv/lib64/python3.5/site-packages/django/core/management/__init__.py", line 327, in execute
    django.setup()
  File "/home/mwenzel/src/OpenSlides/.virtualenv/lib64/python3.5/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/mwenzel/src/OpenSlides/.virtualenv/lib64/python3.5/site-packages/django/apps/registry.py", line 115, in populate
    app_config.ready()
  File "/home/mwenzel/src/OpenSlides/.virtualenv/lib64/python3.5/site-packages/openslides/motions/apps.py", line 15, in ready
    from . import projector  # noqa
  File "/home/mwenzel/src/OpenSlides/.virtualenv/lib64/python3.5/site-packages/openslides/motions/projector.py", line 6, in <module>
    from .views import CategoryViewSet, MotionViewSet, WorkflowViewSet
  File "/home/mwenzel/src/OpenSlides/.virtualenv/lib64/python3.5/site-packages/openslides/motions/views.py", line 28, in <module>
    from .pdf import motion_poll_to_pdf, motion_to_pdf, motions_to_pdf
  File "/home/mwenzel/src/OpenSlides/.virtualenv/lib64/python3.5/site-packages/openslides/motions/pdf.py", line 6, in <module>
    from bs4 import BeautifulSoup
  File "/home/mwenzel/src/OpenSlides/.virtualenv/lib64/python3.5/site-packages/bs4/__init__.py", line 30, in <module>
    from .builder import builder_registry, ParserRejectedMarkup
  File "/home/mwenzel/src/OpenSlides/.virtualenv/lib64/python3.5/site-packages/bs4/builder/__init__.py", line 314, in <module>
    from . import _html5lib
  File "/home/mwenzel/src/OpenSlides/.virtualenv/lib64/python3.5/site-packages/bs4/builder/_html5lib.py", line 70, in <module>
    class TreeBuilderForHtml5lib(html5lib.treebuilders._base.TreeBuilder):
AttributeError: module 'html5lib.treebuilders' has no attribute '_base'
```

**Comment by maweki** [comment]
http://stackoverflow.com/questions/38447738/beautifulsoup-html5lib-module-object-has-no-attribute-base

this is possible the issue. The following restriction possibly solves the issue: `"html5lib<=0.9999999"`

**Comment by emanuelschuetze** [comment]
Right, for install OpenSlides 2.0 you have to install a specifc (old) version of html5lib which does not break our installation. This command works for a fresh install process:
```
pip install openslides html5lib==0.9999999
```

**Comment by maweki** [comment]
You closed this pretty quickly. Is there anything we can do to fix it, instead of having this (pretty bad) workaround?

Is there a way to do it within the requirements-file if there are known bad versions of `html5lib`?

**Comment by emanuelschuetze** [comment]
@maweki Sorry. This issue is already fixed in current master (which is going for 2.1 release). For the 2.0 release we can not fix it and we does not want to make an extra 2.0.1 release (because 2.1-beta1 is coming in the next weeks). So we prefer to use this new install instruction for 2.0 on linux/macOS with this new install comand. Hope this helps.

**Comment by maweki** [comment]
> This issue is already fixed in current master

This is what I wanted to hear. Thank you :)
